spec/classes/collectd_plugin_battery_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper'
+
+describe 'collectd::plugin::battery', type: :class do
+  on_supported_os(baseline_os_hash).each do |os, facts|
+    context "on #{os} " do
+      let :facts do
+        facts
+      end
+
+      options = os_specific_options(facts)
+      context 'default' do
+        it 'defaults' do
+          content = <<EOS
+# Generated by Puppet
+<LoadPlugin battery>
+  Globals false
+</LoadPlugin>
+
+<Plugin "battery">
+</Plugin>
+
+EOS
+
```

Comment:
please also add a `is_expected.to compile.with_all_deps` test

Reply:
done